Dockerfile

```diff
@@ -30,6 +30,7 @@ ENV OPERATOR=/manager \
 
 RUN yum -y install shadow-utils
 RUN useradd -ms /bin/bash nonroot -u 1001
+RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical
```

Comment:
Hi @vivekr-splunk , would you mind explaining the importance of update-minimal here or in the description?

Reply:
I removed minimal as i was testing installing shadow-utils which i was unable to do it , moved to latest that worked. we can move back if needed

Reply:
to only install security errata use `yum update-minimal`